pxr/imaging/hio/OpenEXR/openexr-c.c

```diff
@@ -31,6 +31,12 @@
 // defined but not referenced or exported.
 ARCH_PRAGMA_UNUSED_FUNCTION
 
+#if defined(__clang__)
+#pragma clang diagnostic ignored "-Wmacro-redefined"
+#elif defined(__GNUC__)
+#pragma GCC diagnostic ignored "-Wmacro-redefined"
+#endif
```

Comment:
Which macro is reported as redefined? I didn't see this on linux/mac.

Reply:
I hit most of these with elevated warnings attempting to get a ninja/clang build working on Windows. I can run it again to get an answer as to what triggered it.

Reply:
MSVC 17.7.2, PXR_ENABLE_PYTHON_SUPPORT=OFF (Probably the reason it has not been seen previously)
pxr\imaging\hio\OpenEXR\OpenEXRCore/openexr_conf.h(23,1): warning C4005: 'ILMTHREAD_THREADING_ENABLED': macro redefinition

IlmThreadConfig.h (generated file during the cmake process) has a #define ILMTHREAD_THREADING_ENABLED 1. I'll take a look at whats creating the define during cmake and fix it that way. Was in a rush the first time around :(

Reply:
Not sure if it helps, but the only other occurrence that I saw in my environment was a config file in OpenEXR where they were defining ILMTHREAD_THREADING_ENABLED 1. Also, may no longer be valid - not seeing it in dev.